app/assets/javascripts/application.js

```diff
@@ -27,20 +27,36 @@
 //= require components/editor
 
 $(function() {
-    autosize($('.field textarea'));
-    autosize.update($('.field textarea'));
+	autosize($('.field textarea'));
+	autosize.update($('.field textarea'));
 });
 
 $(function() {
-    if($('[data-choices]').length > 0) {
-        new Choices('[data-choices]', {shouldSort: false});
-    }
+	if($('[data-choices]').length > 0) {
+		new Choices('[data-choices]', {shouldSort: false});
+	}
 });
 
 $(function() {
-    $('.range').range();
-
-    $('.trigger-conditions').on('cocoon:before-insert', function(e, item) {
-        $(item).find('.range').range();
-    });
-});
\ No newline at end of file
+	$('.range').range();
+
+	$('.trigger-conditions').on('cocoon:before-insert', function(e, item) {
+		$(item).find('.range').range();
+	});
+
+	$('.trigger-conditions').on('cocoon:after-insert', function(e, item) {
+		$('select.choose_sensor').on('change', function() {
```

Comment:
`$(item).find('.choose_sensor')` will return only the select field in the item to be added. Otherwise you’re matching all sensor select in the document (which is probably not that severe in most cases, but simply not necessary).